src/runtime/timer_go116.go

```diff
@@ -0,0 +1,32 @@
+//go:build !go1.18
+// +build !go1.18
+
+package runtime
+
+type puintptr uintptr
+
+// Package time knows the layout of this structure.
+// If this struct changes, adjust ../time/sleep.go:/runtimeTimer.
+type timer struct {
```

Comment:
These two files (timer_go116.go and timer_go118.go) seem to be copied from the Go source tree and should therefore have an appropriate notice at the top. Something like this:

https://github.com/tinygo-org/tinygo/blob/f0391eac258ca07ed724526d9f838c4a48333572/src/os/file.go#L1-L3